Backend/app/dashboard/CommonAPIs/spec/RiderPlatform/Management/API/MultiModal.yaml

```
@@ -0,0 +1,79 @@
+    - versionTag: Text
+    - rolloutPercent: Int
+    - derive: "'HideSecrets"
+
+apis:
+  - POST: # PreprocessFRFSDataAPI
+      endpoint: multimodal/frfs/data/preprocess
+      auth: ApiAuthV2
+      request:
+        type: PreprocessFRFSDataReq
+      response:
+          type: PreprocessFRFSDataResp
```

yamllint (1.35.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)